.env.defaults

```diff
@@ -0,0 +1,29 @@
+################################################################################
+#
+# Environment variables for dotenv
+#
+# Copy this file to a new file named .env and fill in the desired parameters.
+# If the variable is set in the environment, it will override the value
+# specified here.
+#
+# Do not commit .env to git.
+#
+################################################################################
+
+# Your API key for Infura
+# Obtain one at https://infura.io/
+# This key is for testing purposes and is not guaranteed to work
+INFURA_API_KEY=bd1da1ddceef40ec9c0d3101e43b3ae6
+
+# Replace this well-known private key with your wallet private key. To export
+# your private key from Metamask, open Metamask and go to
+# Account Details > Export Private Key.
+#
+# Be aware of NEVER putting real Ether into testing accounts.
+#
+# Ropsten faucet: https://faucet.metamask.io/
+# Rinkeby faucet: https://faucet.rinkeby.io/
+# Goerli faucet: https://goerli-faucet.slock.it/
+# Kovan faucet: https://faucet.kovan.network/
+#
+PRIVATE_KEY=0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3
```

Comment:
what is this private key?
From what I know env files should never be part of the public git repo.